modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -87,3 +87,65 @@ Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019 is not su
 . Optional: Back up the `manifests/cluster-network-03-config.yml` file. The
 installation program deletes the `manifests/` directory when creating the
 cluster.
+endif::post-install[]
+ifdef::post-install[]
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Ensure that the cluster uses the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. To configure the OVN-Kubernetes hybrid network overlay, enter the following command:
++
+[source,terminal]
+----
+$ oc patch networks.operator.openshift.io cluster --type=merge \
```

Comment:
Correct patch would be 
```
oc patch networks.operator.openshift.io cluster --type=merge -p '{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":{"hybridClusterNetwork": [{"cidr": "10.132.0.0/14", "hostPrefix": 23}]}}}}}'
```

Note ```[{``` before "cidr"

Reply:
Oops, array of objects.